src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -97,13 +97,17 @@ mutable struct Env
     finalize_called::Bool
     attached_models::Int
 
-    function Env(; output_flag::Int = 1)
+    function Env(; output_flag::Int = 1, memory_limit::Number = 1.0e100)
```

Comment:
```suggestion
    function Env(;
        output_flag::Int = 1,
        memory_limit::Union{Nothing,Real} = nothing,
    )
```

Reply:
What about this option? Then we only set it if the user passes something, which guards against this default changing.